translator/hooks.py

```diff
@@ -103,7 +103,8 @@
 # 		"translator.tasks.daily"
 # 	],
 	"hourly": [
-		"translator.data.write_csv_for_all_languages"
+		"translator.data.write_csv_for_all_languages",
+		"translator.translator.doctype.contributed_translation.contributed_translation.spush_contributions_to_respected_versions"
```

Comment:
typo in "spush_contributions_to_respected_versions", it should be "push_contributions_to_respected_versions"